rust/db_handling/src/identities.rs

```diff
@@ -96,6 +97,63 @@ lazy_static! {
     static ref REG_PATH: Regex = Regex::new(r"^(?P<path>(//?[^/]+)*)(///(?P<password>.+))?$").expect("known value");
 }
 
+#[derive(Encode, Decode)]
+pub enum ExportAddrs {
+    V1(ExportAddrsV1),
+}
+
+#[derive(Encode, Decode)]
+pub struct ExportAddrsV1 {
+    pub addrs: Vec<AddrInfo>,
+}
+
+#[derive(Encode, Decode)]
+pub struct AddrInfo {
+    /// Name of the key
+    pub name: String,
```

Comment:
What is the name? As I see in the Parity Signer UI there is a root key name and others are dervied keys for the specific network. What will be encoded or composed in this parameter?